owned-manifests/machine-api-controllers.yaml

```diff
@@ -36,18 +36,18 @@ spec:
         key: node.alpha.kubernetes.io/unreachable
         operator: Exists
       containers:
-      - name: controller-manager
+      - name: machineset-controller-manager
         image: {{ .Controllers.Provider }}
         command:
-          - "./manager"
+          - /manager
```

Comment:
Was this intentional?

Reply:
yes, to align with all other `command` and other fields syntax